test/parallel/test-async-hooks-worker-asyncfn-terminate-4.js

```diff
@@ -12,6 +12,7 @@ const { Worker } = require('worker_threads');
 const workerData = new Int32Array(new SharedArrayBuffer(4));
 const w = new Worker(`
 const { createHook } = require('async_hooks');
+const { workerData } = require('worker_threads');
```

Comment:
I believe the original author forgot to add this import. And this is why the `strictEqual` below paases, because adding to the `workerData` raises an reference exception.